UPGRADE.md

```diff
@@ -52,10 +89,15 @@ Following Function removed:
 
 The `TagManager` doesn't longer depend on FieldDescriptorFactory, TagEntityName and UserRepository so its constructor changed.
 
+<<<<<<< HEAD
```

Comment:
Solve the conflict :slightly_smiling_face:

Reply:
deja vu 🤔